src/pages/hcp/docs/index.tsx

```diff
@@ -22,6 +22,7 @@ const getStaticProps = generateGetStaticProps({
 	pageContent,
 	productSlugForLoader: 'cloud.hashicorp.com',
 	product: hcpData as ProductData,
+	basePath: 'docs',
```

Comment:
Some `basePath` is now required for custom docs landing server function calls. This allows non-`docs` `basePath` to have custom landing pages.